src/backend/gporca/libgpopt/include/gpopt/operators/CLogicalSelect.h

```diff
@@ -42,6 +42,8 @@ class CLogicalSelect : public CLogicalUnary
 	// table descriptor
 	CTableDescriptor *m_ptabdesc;
 
+	BOOL m_trivial;
```

Comment:
maybe add some commentary, or rename the variable like `m_isInlinedCTE`

Reply:
> add some commentary

done